scipy/stats/tests/test_distributions.py

```diff
@@ -679,6 +679,85 @@ def test_cdf(self):
                                5.06649130e-01, 8.41442111e-01], atol=1e-6)
 
 
+class TestKappa4(TestCase):
+    def test_cdf_genpareto(self):
+        # h = 1 and k != 0 is generalized Pareto
+        x = [0.0, 0.1, 0.2, 0.5]
+        h = 1.0
+        for k in [-1.9, -1.0, -0.5, -0.2, -0.1, 0.1, 0.2, 0.5, 1.0,
+                  1.9]:
+            vals = stats.kappa4.cdf(x, h, k)
+            # shape parameter is opposite what is expected
+            vals_comp = stats.genpareto.cdf(x, -k)
+            assert_allclose(vals, vals_comp)
+
+    def test_cdf_genextreme(self):
+        # h = 0 and k != 0 is generalized extreme value
+        x = [-0.5, -0.2, -0.1, 0.0, 0.1, 0.2, 0.5]
+        h = 0.0
+        for k in [-1.9, -1.0, -0.5, -0.2, -0.1, 0.1, 0.2, 0.5, 1.0,
+                  1.9]:
+            vals = stats.kappa4.cdf(x, h, k)
+            vals_comp = stats.genextreme.cdf(x, k)
+            assert_allclose(vals, vals_comp)
+
+    def test_cdf_genlogistic(self):
+        # h = -1.0 and k != 0 is generalized logistic
+        # EXCEPT there are at least five generalized logistic
+        # distributions.
+        # Four are described here:
+        # https://en.wikipedia.org/wiki/Generalized_logistic_distribution
+        # The "fifth" one is the one kappa4 should match:
+        # https://en.wikipedia.org/wiki/Talk:Generalized_logistic_distribution
+        # http://www.mathwave.com/help/easyfit/html/analyses/distributions/gen_logistic.html
+
+        # The genlogistic currently in scipy doesn't match kappa4, h=-1,
+        # k != 0.
+        pass
```

Comment:
Then this test should be removed IMO, and this point needs to be clarified in the docstring.